echopype/calibrate/ek80_complex.py

```diff
@@ -229,27 +226,21 @@ def get_transmit_signal(
     # Make sure it is BB mode data
     # This is already checked in calibrate_ek
     # but keeping this here for use as standalone function
-    if waveform_mode == "BB" and (("frequency_start" not in beam) or ("frequency_end" not in beam)):
+    if waveform_mode == "BB" and np.all(beam["transmit_type"] == "CW"):
         raise TypeError("File does not contain BB mode complex samples!")
 
     # Generate all transmit replica
     y_all = {}
     y_time_all = {}
+    # TODO: expand to deal with the case with varying tx param across ping_time
+    tx_param_names = [
+        "transmit_duration_nominal",
+        "slope",
+        "transmit_type",
```

Comment:
In theory this can be removed since it is not used in `tapered_chirp`, but I kinda like having the uniqueness check on `transmit_type`. How about adding the below comment?
```suggestion
        "transmit_type",  # not needed for tapered_chirp but good to check uniqueness
```

Reply:
Right! I added it for an initial implementation that used `transmit_type` in `tapered_chirp`, then removed that usage (simplified the code) and forgot to remove the unused `transmit_type`.

Sure, I'll add that comment in the next set of commits.

Reply:
I take this back. `transmit_type` can't remain in `tx_params` b/c the `tx_params` dict is passed to `tapered_chirp`, but `transmit_type` is no longer an argument in that function.